### DISCUSSION TITLE
[TS] No Model Type while creating object

### DISCUSSION THREAD

**Discussion by andrepaulo-bit** [Nozbe/WatermelonDB#1262] [Q&A] [answered]
I created schema, models in TypeScript,
But when I try to create an object, it always says: Property 'serverID' does not exist on type 'Model'.

Whenever I do

```
database.get(TableNames.POST).create(post => {
    post.serverID = p.ID
    post.title = p.Title
})
```

I get the error mentioned before because `post` its a default Model, and not my model `Post`.

Is there anything I can do to specify the model to use?

**Comment by andrepaulo-bit** [comment, marked as answer]
welp, just noticed that I can do `database.get<Post>(TableNames.POST)` and then everything else under that will have the right type.

**Reply by Jenspi** [reply]
Saved my butt all these years later! Thank you so much!
